docs/devops/getting-started.md

```diff
@@ -22,18 +22,28 @@ image:
 ## Best DevOps practices at SAP ##
 
 Continuous delivery is a method to develop software with short feedback cycles.
-It is applicable to projects both for SAP Cloud Platform and SAP on-premise platforms.
-SAP implements tooling for continuous delivery in project "Piper".
-The goal of project "Piper" is to substantially ease setting up continuous delivery in your project using SAP technologies.
+It applies to projects both for SAP Cloud Platform and SAP on-premise platforms.
+SAP implements tooling for continuous delivery in the project "Piper".
+The goal of the project "Piper" is to substantially ease setting up continuous delivery in your project using SAP technologies.
 
 ### SAP Cloud SDK for Continuous Delivery ###
 
 [SAP Cloud SDK for Continuous Delivery](https://github.com/SAP/cloud-s4-sdk-pipeline) is open source and free to use.
-This ready-made pipeline is part of project "Piper" and allows you to start with continuous delivery without writing any pipeline code -- only declarative configuration is required.
-It helps you to quickly deliver high quality applications to SAP Cloud Platform.
+This ready-made pipeline is part of the project "Piper" and allows you to start with continuous delivery without writing any pipeline code -- only declarative configuration is required.
+It helps you to quickly deliver high-quality applications to SAP Cloud Platform.
 
 Check out the [documentation of SAP Cloud SDK for Continuous Delivery](https://sap.github.io/jenkins-library/pipelines/cloud-sdk/introduction/) for more details and usage hints.
 
-### Project "Piper" ###
+## Where to look next? ##
 
-For more information on continuous integration and continuous delivery (CI/CD) in the SAP ecosytem, please refer to the documentation of [project "Piper"](https://sap.github.io/jenkins-library).
+### Learn about Project "Piper" ###
```

Comment:
```suggestion
### Learn about project "Piper" ###
```

official spelling is 'project "Piper"' if it is not at the start of the sentence